### PR TITLE
Labels accessibility for ArrayWidget,SelectWidget,TokenWidget

### PR DESCRIPTION
**Pull request by folix-01**
Readable labels for https://github.com/plone/volto/issues/5139

### CODE REVIEW

**Review by giuliaghisini — APPROVED** [review]

File: packages/volto/news/6332.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Labels accessibility for ArrayWidget,SelectWidget,TokenWidget
```
Comment (Wagner3UB) [inline packages/volto/news/6332.bugfix:1]: add yourself at the end of the comment @folix-01

**Comment by stevepiercy** [comment]
@folix-01 did you recently remove the email from your GitHub account that you used when signing the Plone Contributor Agreement? That's what mr-roboto is nagging you about. We can't merge this until that is resolved.

**Comment by mauritsvanrees** [comment]
@folix-01 Two of the commits have this author:

```
Author: Roman Kysil <romankysil@Romans-MacBook-Pro.local>
```

That may trigger the mr.roboto message.
Or maybe this one as well, which probably happened when you used the GitHub UI to merge the main branch in here:

```
commit ada8a96f9fcd82c38962f56276eb9e8819c2480d
Merge: 6a8bb1f16 a2d997b3e
Author: Roman <72063601+folix-01@users.noreply.github.com>
Date:   Thu Sep 26 11:37:31 2024 +0200

    Merge branch 'main' into 5139
```

Easiest may be to squash all your commits into one and make sure you have the right email on that one.

**Comment by folix-01** [comment]
> @folix-01 Two of the commits have this author:
> 
> ```
> Author: Roman Kysil <romankysil@Romans-MacBook-Pro.local>
> ```
> 
> That may trigger the mr.roboto message. Or maybe this one as well, which probably happened when you used the GitHub UI to merge the main branch in here:
> 
> ```
> commit ada8a96f9fcd82c38962f56276eb9e8819c2480d
> Merge: 6a8bb1f16 a2d997b3e
> Author: Roman <72063601+folix-01@users.noreply.github.com>
> Date:   Thu Sep 26 11:37:31 2024 +0200
> 
>     Merge branch 'main' into 5139
> ```
> 
> Easiest may be to squash all your commits into one and make sure you have the right email on that one.

Thank, just tried, it works

File: packages/volto/news/6332.bugfix [hunk 2]
```diff
@@ -0,0 +1 @@
+Labels accessibility for ArrayWidget,SelectWidget,TokenWidget. @folix-01
```
Comment (pnicolli) [inline packages/volto/news/6332.bugfix:1]:
```suggestion
Labels accessibility for ArrayWidget, SelectWidget, TokenWidget. @folix-01
```

**Review by pnicolli — APPROVED** [review]

**Review by JeffersonBledsoe — APPROVED** [review]
Labels now actually have the correct role and the inputs associate the name from the label. Looks good!

We also have the same problem with the file widget (e.g. preview image field) and the checkbox widget (e.g. Exclude from navigation), but we can always open another PR for them if you don't want to include them here

**Comment by pnicolli** [comment]
Yeah I would merge this one in its current state. Are there issues already for the other ones? @JeffersonBledsoe

**Comment by JeffersonBledsoe** [comment]
> Are there issues already for the other ones? @JeffersonBledsoe 

Not yet, they're implied in the linked issue. I'll write explicit issues for them shortly

**Pull request merged**